pytorch_pipeline/examples/cifar10/cifar10_pytorch.py

```diff
@@ -114,7 +113,7 @@
 trainer = Trainer(
     module_file="cifar10_train.py",
     data_module_file="cifar10_datamodule.py",
-    module_file_args=parser,
```

Comment:
@shrinath-suresh Have we verified this change?

Reply:
Yes Geeta. All the other arguments are of type `dict`. Only the module_file_args is passed as `ArgumentParser`. I changed it to `dict` to keep it uniform. 

And also ran the scripts locally to verify the examples.